### ISSUE TITLE
It should not be possible to merge the same word that you are currently on.

### ISSUE THREAD

**Issue by interestinall** [wordset/wordset#103]
Let's say you are on the word color. Now you can "merge" color with color and therefore delete color. This could be particularly troublesome when there is only one word and someone is looking to add the second but makes a thinking mistake.

**Comment by HamptonMakes** [comment]
hahahhaa... yowza.

**Comment by postpostscript** [comment]
I've added the validation messages.

An issue that I found with this is is that error messages _do not_ disable the submit button on both the add and edit proposal pages. I can work on that as part of this issue, or I can add another issue. Whichever you/y'all prefer.

**Comment by HamptonMakes** [comment]
Yeah, do that as a different issue. We'll sort it out before pushing this live.
